src/app.ts

```diff
@@ -102,10 +108,20 @@ async function start() {
     try {
         app = new App();
         await app.ready;
-        console.log('app started');
+        console.log(timestamp`app started`);
     } catch (err) {
-        throw new Error(`App start failed ${err}`);
+        throw new Error(`app start failed ${err}`);
     }
 }
 
-start();
\ No newline at end of file
+start();
+
+
+/*
```

Comment:
Why commented out? If debug code should this still be on branch? if functional shouldn't we just include?

Reply:
Decided to remove debug.  But there is a little bit of tricky config.  Until confirmed wanted to keep it handy.